modules/core/css/index.scss

```diff
@@ -7,5 +7,5 @@
 @import './lib/accessibility.scss';
 
 // Components
-@import './lib/components/icon-list.scss';
+@import '~@workday/canvas-kit-css-icon/lib/icon-list';
```

Comment:
Question: if we're moving it from `core` to `icon` isn't this line supposed to be deleted rather than modified? Or was there a specific reason we need to keep this in core?

Reply:
Oh wait, I see. I guess `core` relies on this.

Reply:
yep yep!

Reply:
@lychyi had it right here. Everything related to `icon-list` in core should be moving into the icon module. Everything in `core/css/lib/components/accessibility.scss` now pertains to icon-list and should just be moved over. We're trying to completely eliminate the `components` folder from `core/`

Reply:
oh i see! I'll make the change!